layers/+source-control/version-control/packages.el

```diff
@@ -54,7 +54,8 @@
       ;; If you enable global minor mode
       (when (and (eq version-control-diff-tool 'git-gutter)
                  version-control-global-margin)
-        (global-git-gutter-mode t))
+        (run-with-idle-timer 5 nil (lambda ()
```

Comment:
And then move the `(global-git-gutter+-mode t)` to config? It'll still run init right away, right?

That seems like a good idea/use of use-package.